### DISCUSSION TITLE
.NET MAUI support

### DISCUSSION THREAD

**Discussion by gianox** [thudugala/Plugin.LocalNotification#241] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
NotificationCenter.Current.Show does not work in .NET MAUI

**Describe the solution you'd like**
NotificationCenter.Current.Show works in .NET MAUI

**Additional context**
Only tested on Android

**Comment by thudugala** [comment]
@gianox will look in to this when Visual Studio 2022 has good direct support for .NET MAUI.

**Comment by thudugala** [comment, marked as answer]
@gianox Try Version 10.0.0
